cidc_api/config/settings.py

```diff
@@ -27,6 +27,8 @@
 TESTING = environ.get("TESTING") == "True"
 ALLOWED_CLIENT_URL = environ.get("ALLOWED_CLIENT_URL")
 IS_GUNICORN = "gunicorn" in environ.get("SERVER_SOFTWARE", "")
+# DO NOT CHANGE - used only for mocking in tests that need to bypass this feature
+DISABLE_METADATA_VALIDATION_ON_INSERT = False
```

Comment:
This doesn't seem like the best way to me; seems kind of hacky to have functionality that's only needed for testing

Reply:
agreed
@jacoblurye you can monkey patch the code while in tests to circumvent the validation.